static/html/adminmenu.htm

```diff
@@ -1,15 +1,15 @@
 <div class="menu">
   <h3><u>Menu</u></h3>
-  <p><a href="/">Home</a></p>
-  <p><a href="/commands">Commands</a></p>
-  <p><a href="/stats">Stats</a></p>
-  <p><a href="/history/">Songlist History</a></p>
+  <p><a href="/">• Home</a></p>
```

Comment:
You should probably do this in CSS. You can do this using the `:before` pseudo elements.

The HTML:
```html
<aside class="menu">
  <h3 class="menu_title">Menu</h3>
  <p class="menu_item">
    <a href="/">Home</a>
  </p>
</aside>
```

```css
.menu_item:before {
  content: '• ';
}
```

This will add the content `'• '` before the `.menu_item` element. This will be better for screen readers and Google. You might want to read about semantic html5. A `<p>` element is theoretically not for menu items, they are for paragraphs of text.

The better semantic navigation would look like this:

```html
<aside class="sidebar">
  <section class="menu">
    <h3 class="menu_title">Menu</h3>
    <nav>
      <ul>
        <li><a href="bla">Home</a></li>
      </ul>
    </nav>
  </section>
</aside>
```